src/routes/landing/components/LocationSelector.css

```diff
@@ -10,7 +14,7 @@
     color: inherit;
     text-decoration: inherit;
     transition: background-color 0.5s ease;
-    background-color: var(--hover-color);
+    background-color: var(--background-accent-color);
```

Comment:
this color is a bit lighter than the one we used before (rgba(0, 0, 0, 0.1)), but since we agreed to only use some defined colors this should be ok